extensions/integration-tests/src/test/schemaCompare.test.ts

```diff
@@ -214,6 +217,7 @@ suite('Schema compare integration test suite @DacFx@', () => {
 			await utils.tryDeleteDB(server, targetDB, ownerUri);
 		}
 	});
+
 	test('Schema compare dacpac to dacpac comparison with include exclude', async function () {
 		this.timeout(5 * 60 * 1000);
```

Comment:
probably better to create a constant for timeout to use in all tests. Also 5 minute might not be enough